changelogs/unreleased/gh-9556-fix-uri-ipv6.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/uri
```

Comment:
Nit: This way `Uri` becomes a top-level subsection in the `Bugs fixed` section. As for me, it should be in the `Lua` subsection.

IOW, I would make it `## bugfix/lua/uri`.

Reply:
replaced